crime-means-assessment/build.gradle

```diff
@@ -81,6 +95,15 @@ test {
     finalizedBy jacocoTestReport
 }
 
+cucumber {
+    main = 'io.cucumber.core.cli.Main'
+    glue = 'classpath:uk.gov.justice.laa.crime.meansassessment.cucumber'
+    plugin = ['pretty']
+    featurePath = 'src/test/resources/uk/gov/justice/laa/crime/meansassessment/cucumber'
+}
+
+tasks["build"].dependsOn tasks["cucumber"]
```

Comment:
Should this extend test instead?

Reply:
I think this works - does it need to be changed?

Reply:
I quite often choose to skip running the tests whilst I'm iterating e.g. 

`./gradlew clean build -x test`

Tying Cucumber to the build step would make this a little more difficult, but it's not the end of the world, it just having to exclude another task.

Reply:
do you have an example I can copy? [This was just copied from an example I found]